db/rdb/src/main/java/it/unibz/inf/ontop/model/term/functionsymbol/db/impl/DuckDBDBFunctionSymbolFactory.java

```diff
@@ -162,6 +162,11 @@ public DBBooleanFunctionSymbol getDBRegexpMatches2() {
         return (DBBooleanFunctionSymbol) this.regexpLikeFunctionSymbol;
     }
 
+    @Override
+    public DBBooleanFunctionSymbol getDBRegexpMatches3() {
+        throw new UnsupportedOperationException("DuckDB Regex matches with three arguments is not supported by Ontop.");
```

Comment:
This message is misleading: we don't know how to translate the Ontop expression into DuckDB dialect, not the other way around